croissant-rdf/notebooks/networkx.ipynb

```diff
@@ -0,0 +1,426 @@
qninjVYaI0nKvomaJ39I8Pv99PX1qVOUvJKGPGusVqtKkEWnuLi4EftPlmVqamqoqKhAlmXmzp1LaWnpqMdAMBhU/cX6+/vV5263G0mSCAaDqum9VqvFYDAQDAZJSkpS/cWi6328oX+UcIv+xyaT6QQ1mcPh4NChQ2rZ1vTp07GebgnssO3t7u5Wy6b6+/sRRVHtLNvtdjZu3IgsyyxfvpxwOMzevXupr68nEolgs9kwGo0jFJ1R37IoURYXF0dvby8NDQ1YLBZKS0upra2lzZyBlFGCGA4gSxEsFgsWswUZGeeAk4gUIT4+Ho2oHE8yMoOuQYLBAFabDYPeQCAi4Q5JRNTWi4w0VGJ1NvdFSQaTVsCq06j7JzpQZCGMZ99GPC21pKSkMHHiRMrKyk4wlI8SQtdcc436WX9/Py+99JJKwrvdbjIyMpg8eTLp6ekcOXKEgwcP4vF4yM7Opry8nKysLGRZpq2tjdraWhobGwmFQqSmplJcXExhYeEJycN1dXW88847ZGdn09XVxfjx4zl69CjOoRROrVZLcXEx06ZNO6GEUpIk+vr61PLHhoYGenp60Ov1ZGdnk56eTk9PD06nk89//vOjph5LksTq1avR6XRccskl6n/QPBhiV4+PTl+YkCSjEwT0IgwMONDr9RiNRgYGBpCGjiGLJe44MkTG6w/gCQSREJEjYXLtJhbmJY5QAERDDmbNmsWkSZN488036e7uZtmyZezduxeHw8HKlStHEJWSJHHgwAG2bNmC2+3GarWyaNEiysrK6O3tZceOHbS1tZGamsqsWbPIysoClLLZ9957j76+PsrLy5kxY4ZKCg7/LKoKixKcfX19rF+/nr6+PoxGI4FAgLy8vA/UJywa5tDY2DiC9M7NzSUvL4/s7Gx0Op2qbm1oaKCrqwtRFFWVn9VqVdWK0UAPt9utnufTpk1jxowZZ61ii6ZARr3LfD4f8fHxY6c6xxBDDJ8YdPvCvFjvQoOAXvPJO1+DEZkIMlcW2kg1fcLsVmL4RCNGiP0XYbTGpUEjnFEppSTLBCKyWhqSbtIyI9UUk5fG8KlFJBI5LeLM7/cjyzLp6elccsklZ/+DknSMIIuem8PVW1u2KETShRcei5kfDQ6HQpa9/PIIsmnUWHmAf/xDMQZ+7DF4+GF46y2lNGTvXsUY+NJLobhYUaqlpSlEVfTyf6YdkR074KtfVeLkzz1Xee+VV5Rlrlr1garEZFlmYGCA+Pj40+4wRSIRBgYGRpBkfX19+P1+QCnDTUxMHEGSJSQkEAqFqKiooKamhoyMDBYvXjxqkt5oGE6UORwOurq66OjowOl0qmVwUdVVWVkZeXl5xMfHk5CQMMInLxwOj5l+GU29FEVRJVgACgsLmT59OmlpaWfluTc4ODiinEqSJOx2u+pBtnLlSnJycvD7/VRVVXHw4EFCoRBFRUWUlpYiiqJq8u9yuRgcHMTpdBIMBlX1XDgcxpxdRGjcPDSiQMDjBlnGZDIRZ7WiETVIssSAYwBBQCEJBAGXy0UoGBwKA9Djiyjl/bI8sjQy2mE/G4+hqJ9YvF6DXhSQZQl/IIDX5ycsapEjYcZpvayaWjKm4rC+vp63336bG2+8EYPBQFdXFy+88AJer5f4+HiKi4uZPHkyBoOBAwcOcOTIESRJUt9PTEwcdbnhcJimpiZqa2tpaWlBlmUyMzMpLi6moKAAvV6P3+/nr3/9K7Iss2LFCnw+H/v27WNgYABBENBoNBQUFDB16lQSEhLo6elRvb+6u7sJh8NoNBp0Oh39/f3k5uZywQUXYLFYVKJv8eLFjB8/ftR1rKqqYuvWrVx22WWkpKScdBDP4/Xg8/qw2W24nC4ikTCiRoOAQGJSIqKgXDtC4ZB63BgMBswmMyEZwgjqIN6cNBOuvl5effVVCgsLWbBggZoMumzZMvbs2YPD4WDVcUmiXV1drF+/ntbWVjQaDZMmTWLhwoUEg0EqKytpaGggISGBmTNnqt5qUT+wffv2ER8fz5IlS9RlSpLEvn372LVrF3a7naVLl6qfybLM3r171TRagOTkZObNm6eWe74f+Hw+9dxta2sjHA5jt9vJz88nLy+PtLQ0VcFYXV1NQ0MDfX19yLKM2WxGr9erBH4UOp1uhFn98Qb2Z3qNid6zAoEATU1N6jmRk5Pz3xuuE0MM/4PY2O5hb58f6ydMHRaFLMsMhiWmJhlZHPMSi+EMECPE/sswWvlBNENLIwhoBKXRKTDUyJdlIjIjyg+i5sETEgwkG89uRD2GGD5tGB4QoNfrsdvtY5NPZ4POTsXM/rzzFG+xhx6CvLzRyyJlWUmfdLvhK185veWvWqWouf7+d+V3iouVsszbb4eJExWibMUKZd5wGFpbldLK4uIzJ7AeeEAh2379a0Uddvvt0NYGF12krO/Bg/Dqq1BWBpdcomzvWfqjRSIRKioq2LFjBzk5OYwbN05NTjyTzlS07Od4kszpdKr/czS1TxAEmpqaCIfDairc2XbcgsEgLS0tVFdXU1dXp5JMOp2OuLg4DAaDWjIZ9SEbjSiLmuYPJ8n6+/tpbW3F5XKpHdyUlBR1GcOn0y0VDgaDtLa2qp3snp4eIpEIJSUlzJw5U1UwVVdXs3//fjwej5pMObwcTZZlAoGAmoRZ39DAfiGZiDUZ/B5kFGJPlpRUY51Wi06vRxQEfH4/Wq0WURAJh8PY7DZ0Wh3eITKM48gwUJRkkfD7U4lpRdCFfKpnncFgwGgwEhY1yAjMPUnYzOHDh9m0aROrVq3iwIEDHDhwAFmWmT9/PtOnT2dwcJD9+/fT2NiIwWBg4sSJTJw4cVTF1VgIBALU19dTW1tLZ2cnoiiSm5tLRkYGa9euVf3hQqEQpaWllJeXA7B3714OHz6My+VSk4DtdjtpaWlq+WN3dzfbtm1j/PjxLFq0SCWBXnzxRQwGwwjl13D4/X7+/e9/k5+fz5IlS2gaDLKpw4sjEEF/nM2DJEv09/crqslAkHA4
```

Comment:
Exception

Reply:
As in add an exception for the CI linter?

Reply:
Sorry David! My comment was very blunt. I meant the code raises an exception - is that excepted?